### PR TITLE
Fix _resizeEvent being called multiple time issue

### PR DESCRIPTION
**Pull request by pandamicro**
> Makes sure these boxes are checked before submitting your PR - thank you!
> - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
> - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)
> - [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
> - [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.

Re: cocos-creator/fireball#2555

Changes proposed in this pull request:
- 使用另一种方式解决微信闪屏的问题

@VisualSJ 麻烦明天测试一下是否可行
@cocos-creator/engine-admins

### CODE REVIEW

**Comment by pandamicro** [comment]
Related https://github.com/cocos-creator/engine/pull/560

File: cocos2d/core/platform/CCView.js [hunk 1]
```diff
@@ -210,6 +213,7 @@ var View = cc._Class.extend({
         if (view._resizeCallback) {
             view._resizeCallback.call();
         }
+        this._resizeCalled = false;
```
Comment (jareguo) [inline cocos2d/core/platform/CCView.js]: 建议加点注释……

**Comment by VisualSJ** [comment]
@pandamicro 
在iphone 6 plus上测试微信浏览器不会黑屏了。
但是还是和之前的修改一样，会闪屏一下。

**Comment by pandamicro** [comment]
就闪一下就好了？

**Pull request merged**